docs/en/latest/getting-started.md

```diff
@@ -155,23 +161,25 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
 }'
 ```
 
-This routing configuration means that all matching inbound requests will be forwarded to the Upstream service `httpbin.org:80` when they meet **all** the rules listed below:
+This configuration basically means that it will forward all matching inbound requests to the upstream service (`httpbin.org:80`) if they meet these specified criterions.
```

Comment:
```suggestion
This configuration means that it will forward all matching inbound requests to the upstream service (`httpbin.org:80`) if they meet these specified criteria.
```

Reply:
criteria --> criterion in plural?